.github/workflows/test-integrations-ai.yml

```diff
@@ -39,7 +39,7 @@ jobs:
           allow-prereleases: true
       - name: Setup Test Env
         run: |
-          pip install coverage tox
+          pip install "coverage[toml]" tox
```

Comment:
What's the difference to plain `coverage`?

Reply:
In python 3.6 you need the extra or it can not read the pyproject.toml file...